### PR TITLE
Revised and turned back on integration tests [stage-2]

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review the last commit, the others are from previous PR #121 yet to be merged. Thanks

File: test/integration/logging-storage-file.html [hunk 1]
```diff
@@ -86,6 +89,8 @@
 
       test("should log the configuration event", function () {
 
+        console.log(spy.args);
```
Comment (donnapep) [inline test/integration/logging-storage-file.html]: @stulees Should this be here?

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks. I tried to improve upon our unit tests but I could only more or less equal what we had previously. It would require an elaborate mock of jwplayer to improve on them. What's your thoughts?

**Comment by donnapep** [comment]
👍

**Pull request merged**